demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementFromRowBuilderWithZipSupport.php

```diff
@@ -69,18 +80,25 @@ public function setOriginalFileReferences(Cell $cell): ?ConstraintViolationListI
         return null;
     }
 
-    public function setFileReferences(Cell $cell): ?ConstraintViolationListInterface
+    public function setOriginalFileReferences(Cell $cell): ?ConstraintViolationListInterface
+    {
+        $this->originalFileReferences = $cell;
+
+        return null;
+    }
+
+    protected function handleFileReferences(): ?ConstraintViolationListInterface
     {
         // early return in case no file-reference is found
-        $cellValue = $cell->getValue();
+        $cellValue = $this->fileReferences->getValue();
```

Comment:
`$cellValue` needs to be validated if it is actually a string. Otherwise we may trigger exceptions instead of collecting violations as intended.

Reply:
This simply skips the value. Please use [Symfony Validation](https://symfony.com/doc/current/validation.html).

Reply:
done in [a960b74](https://github.com/demos-europe/demosplan-core/pull/2575/commits/a960b749db2f368ae8a416e655bcd47b1e9240b1)

Reply:
for history: we did need the validation